tests/test-cases/Group9-VIC-Admin/9-01-VICAdmin-ShowHTML.robot

```diff
@@ -110,6 +110,13 @@ Get Container Logs
     ${vmName}=  Get VM Display Name  ${container}
     Should Contain  ${output}  ${vmName}/vmware.log
     Should Contain  ${output}  ${vmName}/tether.debug
+    ${rc}  ${output}=  Run And Return Rc and Output  curl -sk %{VIC-ADMIN}/container-logs.tar.gz -b /tmp/cookies-%{VCH-NAME} | (cd /tmp; tar xvzf - ${vmName}/tether.debug)
```

Comment:
Why are we downloading the logs a second time? The logs have already been downloaded and extracted on line 107.